src/language-js/index.js

```diff
@@ -10,7 +10,15 @@ const languages = [
     override: {
       since: "0.0.0",
       parsers: ["babylon", "flow"],
-      vscodeLanguageIds: ["javascript"]
+      vscodeLanguageIds: ["javascript"],
+      shebangs: [
```

Comment:
Fair enough! I was simply unaware of that. Though I know that some package managers will install Node as `nodejs` rather than just `node`, but I presume that *extending* the `interpreters` field is OK?

I suppose, in this case, that you come down in favour of parsing the shebang to extract the interpreter rather than comparing the shebang as a whole.

Reply:
Extending is fine, just use the `extend` field from `createLanguage`:

https://github.com/prettier/prettier/blob/42572f26acbf6c6c640b5cdfb69f202ceaea3c5d/src/language-js/index.js#L71-L80

---

> I suppose, in this case, that you come down in favour of parsing the shebang to extract the interpreter rather than comparing the shebang as a whole.

Yes, since we don't care the rest parts.

Reply:
Added a stab at this.